### ISSUE TITLE
How to use the plugin with vue-i18n?

### ISSUE THREAD

**Issue by yfwz100** [underfin/vite-plugin-vue2#65]
Can anyone provide an example?

**Comment by yfwz100** [comment]
If #66 is merged, a simple `vue-i18n` adapter plugin can be specified in the plugins of vite to parse i18n sfc resource.

e.g.

```js
import yaml from 'js-yaml';

function createVueI18nPlugin() {
  const i18nSrcParser = (source) => JSON.stringify(yaml.load(source)); // parse source as YAML by default.
  return {
    name: 'vite-plugin-vue2-i18n',
    transform(source, id) {
      if (/type=i18n/i.test(id)) {
        return {
          code:
            `const __i18n =${i18nSrcParse(source)};` +
            `export default function i18n(Component) {\n` +
            `  const options = Component.options || Component\n` +
            `  options.__i18n = options.__i18n || []\n` +
            `  options.__i18n.push(JSON.stringify(__i18n))\n` +
            `}`,
          map: {
            mappings: '',
          },
        };
      }
    },
  };
}
```

**Comment by yfwz100** [comment]
I've written a plugin for this issue. See  https://www.npmjs.com/package/@yfwz100/vite-plugin-vue2-i18n for more info.

**Comment by underfin** [comment]
fixed at #66
